src/parser.js

```diff
@@ -108,7 +108,12 @@ class Parser {
           pugNode += `.${value.split(' ').join('.')}`
           break
         default:
-          attributeList.push(`${name}='${value}'`)
+          if (!value) {
+            attributeList.push(`${name}`)
```

Comment:
This is great! If you could rebase with `develop` branch and include this conditional, I'd be happy to merge it in

Reply:
Added in 075d462cd0d104e7f8dfa98e231213b1ea6b541d